src/testIntegration/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/onetimecode/OneTimeCodeResourceTest.kt

```diff
@@ -0,0 +1,152 @@
+package uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.onetimecode
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.MediaType
+import org.springframework.web.reactive.function.BodyInserters
+import org.springframework.web.reactive.function.client.WebClient
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.IntegrationTest
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.EmailInvalid
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.EmailInvalidCjsm
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.EmailMandatory
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.MalformedRequest
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.SessionIdMandatory
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.magiclink.Message
+
+class OneTimeCodeResourceTest(
+  @Value("\${mailcatcher.api.port}") private val mailcatcherApiPort: Int,
+) : IntegrationTest() {
+
+  private val mailCatcherWebClient = WebClient.builder().baseUrl("http://localhost:$mailcatcherApiPort").build()
+
+  @Nested
+  inner class CreateMagicLink {
```

Comment:
typo

Reply:
Good spot , I'll fix that in my next PR 👍